### DISCUSSION TITLE
How to make it work with AMD on Windows?

### DISCUSSION THREAD

**Discussion by dveblck** [lllyasviel/stable-diffusion-webui-forge#662] [Q&A]
The title says it all.

**Comment by RobertoMaurizzi** [comment]
You need to wait for AMD and PyTorch to have a PyTorch-rocm for Windows. It's not currently available, not even for the nightly version.
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/451686/7952e5c0-8273-405f-9b76-7c2dee333af7)
